bin/merge.sh

```diff
@@ -90,6 +90,12 @@ substitute_file_content_40s_into_40x () {
 
   echo "=== Exchange 40x/X with 40s/S in file content ==="
 
+  read -p "Exchange 40x/X with 40s/S in file content? y/n " yn
+  case $yn in
+    [Yy]* ) ;;
+    * ) echo "Skip content substitution"; return;;
+  esac
```

Comment:
So this is "opt in"?
Defaults to _not_ substituting?

Maybe the y/n message should say something about the default, hinting at the default being a safe recommendation?
`(default: n)` or something short and user-friendly, so people don't have to stop and think.

Reply:
Good point. I will fix it in another PR.